change/@fluentui-react-checkbox-f702f678-e790-45fc-88a7-06ff6ed30e2e.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "refact: remove mergeprops from react-checkbox",
```

```suggestion
  "comment": "refactor: remove mergeProps usage",
```

As this will go to a changelog, let's have a better message there

---

@bsunderhus it seems that something went wrong and this change is not applied...